lib/Agrammon/Model.pm6

```diff
@@ -375,10 +375,10 @@ class Agrammon::Model {
     #| string names of their taxonomies. A multi-instance module root is a pair,
     #| where the key is the taxonomy name of the root of the multi-instance module
     #| and the value is an array of the modules within that instance.
-    method extract-structure() {
+    method extract-structure {
         my @singles;
         my %multis;
-        for @!evaluation-order -> Agrammon::Model::Module $module {
+        for @!evaluation-order.reverse -> Agrammon::Model::Module $module {
```

Comment:
This bit probably needn't change if it's not being used for the inputs?

Reply:
Reverted